kclvm/sema/src/advanced_resolver/node.rs

```diff
@@ -867,6 +867,42 @@ impl<'ctx> MutSelfTypedResultWalker<'ctx> for AdvancedResolver<'ctx> {
     }
 
     fn walk_config_expr(&mut self, config_expr: &'ctx ast::ConfigExpr) -> Self::Result {
+        let (start, end) = (self.ctx.start_pos.clone(), self.ctx.end_pos.clone());
+        let schema_symbol = self.ctx.schema_symbol_stack.last().unwrap_or(&None).clone();
+        let kind = LocalSymbolScopeKind::Config;
+
+        self.enter_local_scope(
+            &self.ctx.current_filename.as_ref().unwrap().clone(),
+            start.clone(),
+            end.clone(),
+            kind,
+        );
+
+        let cur_scope = *self.ctx.scopes.last().unwrap();
+        self.gs
+            .get_scopes_mut()
+            .set_owner_to_scope(cur_scope, schema_symbol.unwrap());
+
+        if let Some(schema_expr) = self
+            .gs
+            .get_symbols_mut()
+            .exprs
+            .get_mut(schema_symbol.unwrap().get_id())
```

Comment:
Try to minimize the use of unwrap() in Rust. `schema_symbol` may be None, but at line 890 `.get_mut(schema_symbol.unwrap().get_id())`, it is very easy to panic.